packages/augur-ui/src/modules/create-market/components/common.tsx

```diff
@@ -26,4 +34,50 @@ export const Subheaders = (props: SubheadersProps) => (
       {props.link && <a target="blank" href="https://docs.augur.net">Learn more</a>}
     </span>
   </div>        
+);
+
+export interface SmallSubheadersProps {
+  header: string;
+  subheader: string;
+}
+
+export const SmallSubheaders = (props: SmallSubheadersProps) => (
+  <div className={Styles.SmallSubheaders}>
+    <span>{props.header}</span>
+    <span>
+      {props.subheader}
+    </span>
+  </div>        
+);
+
+export interface ExplainerBlockProps {
+  title: string;
+  subtexts: Array<string>;
+}
+
+export const ExplainerBlock = (props: ExplainerBlockProps) => (
+  <div className={Styles.ExplainerBlock}>
+    <span>
+      {props.title}
+    </span>
+    {props.subtexts.map(subtext => 
+      <span>
```

Comment:
and assuming this is actually a paragraph style subtext then use a p tag, otherwise the span is fine if it's just like one sentence.